### PR TITLE
feat: :technologist: Added math library

### PR DESCRIPTION
**Pull request by jksevend**

### CODE REVIEW

File: packages/tobot_math/tests/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/03597a01ee50ed33e9dfd640b249b4be3799d395.zip
+)
+
+# For Windows: Prevent overriding the parent project's compiler/linker settings
+set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+FetchContent_MakeAvailable(googletest)
+
+set(TOBOT_MATH_TESTS tobot-tooling_TESTS)
```
Comment (FrederikTobner) [inline packages/tobot_math/tests/CMakeLists.txt]: Why tooling test 🗡️

Comment (FrederikTobner) [reply packages/tobot_math/tests/CMakeLists.txt]: Additionally adapt the name in the test.yaml file

**Review by FrederikTobner — APPROVED** [review]
lgtm

File: packages/tobot_math/tests/CMakeLists.txt (hunk 1 above)
Comment (jksevend) [reply packages/tobot_math/tests/CMakeLists.txt]: Fixed with https://github.com/jksevend/tobot-engine/pull/47/commits/0ec2e6c749992ec5fbaa43cbc8e8822871ed09ff

**Pull request merged**